magda-semantic-indexer-framework/src/config.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I guess it's a default config file --- if so, could we rename it to default-config.json to make it more obvious?